src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const formatString = sourceString.replace(/;\s*;/g, ' ').trim();
```

Comment:
[CODE KNOWLEDGE] - You don't need to make a copy before each operation. The `replace` method does not mutate the original string, so you can chain other methods directly without needing to assign it to `formatString`.